pandas/tests/indexes/test_indexing.py

```diff
@@ -0,0 +1,84 @@
+"""
+test_indexing tests the following Index methods:
+    __getitem__
+    get_loc
+    get_value
+    __contains__
+    take
+    where
+    get_indexer
+    slice_locs
+    asof_locs
```

Comment:
why are we not using tests/indexes/methods/test_* like the frame and Series tests.